menu.py

```diff
@@ -0,0 +1,31 @@
+import pygame 
+
+import sys
```

Comment:
The order of imports should be, 
built-in
3rd party
your own

So in this case, it should be.
`import sys`
`import pygame`
`from gamemaster mport GameMaster`